packages/mds-provider-service/repository/entities/mds-provider-entity.ts

```diff
@@ -0,0 +1,26 @@
+import { IdentityColumn, RecordedColumn } from '@mds-core/mds-repository'
+import { Nullable } from '@mds-core/mds-types'
+import { Column, Entity } from 'typeorm'
+
+@Entity('mds_providers')
+export class MDSProviderEntity extends IdentityColumn(RecordedColumn(class {})) {
+  @Column('uuid', { primary: true })
+  provider_id: string
+
+  @Column('varchar', { length: 255 })
+  provider_name: string
+
+  @Column('varchar', { length: 255, nullable: true })
+  url: Nullable<string>
+
+  @Column('varchar', { length: 255, nullable: true })
+  mds_api_url: Nullable<string>
+
+  @Column('varchar', { length: 255, nullable: true })
+  gbfs_api_url: Nullable<string>
+
+  @Column('varchar', { length: 7, nullable: true })
+  color_code_hex: Nullable<string>
+}
```

Comment:
I think there was recently some mode attributes added to Provider config. Might be cool to incorporate that here as well.